app/app/src/main/java/com/morning5/vocabularytrainer/database/VocabularyData.java

```diff
@@ -35,4 +38,24 @@ public String getWord2() {
     public String getLanguage2() {
         return language2;
     }
+
+    public JSONObject toJSON() {
+
+        JSONObject jsonObject = new JSONObject();
+
+        try {
+            jsonObject.put("id", getId());
+            jsonObject.put("word1", getWord1());
+            jsonObject.put("language1", getLanguage1());
+            jsonObject.put("word2", getWord2());
+            jsonObject.put("language2", getLanguage2());
```

Comment:
would've liked a more generic approach here, since there will be two more columns later on (rating and custom tags)